msg/DisplayRobotState.msg

```diff
@@ -3,3 +3,7 @@ RobotState state
 
 # Optionally, various links can be highlighted
 ObjectColor[] highlight_links
+
+# Optionally, hide the robot state if true.
+# This is useful in Rviz to programmatically toggler robot visualization
+Bool hide_robot_state
```

Comment:
This message is all about RobotState. So, why adding `robot_state` as a suffix to this variable name?
```suggestion
Bool hide
```
Alternatively I suggest `hide_display`

Reply:
I think ``hide`` is too short a variable name. I like hide_display
```suggestion
Bool hide_display
```